tests/pass-dep/libc/gettid.rs

```diff
@@ -0,0 +1,9 @@
+//@only-target-linux
+
+use libc::{getpid, gettid};
+use std::thread;
+
+fn main() {
+    thread::spawn(|| assert_eq!(unsafe { gettid() }, 1001));
```

Comment:
What is this testing? Or rather, why are we checking that gettid returns this specific value?

Reply:
I am also not sure if assertion provides much value. I wanted to test that `gettid` will always return the same value in isolation mode, since determinism is expected. I don't have strong feelings about it, I can remove it if you feel this isn't wanted.

Reply:
That's a fair idea, but I wouldn't mind a comment that explains that the only point is to ensure that we always return the same value, and that the specific value 1001 is not important per se.

Reply:
done!

Reply:
actually I added a similar comment in the `getpid_with_isolation`  test.